src/customFields.gs

```diff
@@ -158,43 +158,15 @@ function saveCustomFields(jsonFormData) {
  * @desc Fetch list of all Jira fields (name and id) and show them in a sidebar.
  */
 function sidebarJiraFieldMap() {
```

Comment:
I needed to fetch all of the fields for the feature so I moved this code into a single re-usable method

Reply:
The Jira Field Map feature needs testing - I have done so and it seems ok but I'd appreciate you helping with that in case I missed something.

Reply:
https://github.com/ljay79/jira-tools/issues/140